src/tests/topp/FeatureFinderIdentification_1_output.featureXML

```diff
@@ -4275,10 +4275,10 @@
 			<userParam type="float" name="var_norm_rt_score" value="0.00189903781782697"/>
 			<userParam type="float" name="var_intensity_score" value="0.449663067292516"/>
 			<userParam type="float" name="nr_peaks" value="3"/>
-			<userParam type="float" name="sn_ratio" value="1.18492061027865"/>
-			<userParam type="float" name="var_log_sn_score" value="0.169675776797353"/>
+			<userParam type="float" name="sn_ratio" value="21.7975863988899"/>
```

Comment:
Okay, this is indeed a huge change (signal-to-noise ratio from 1.2 to 21.8). Any idea why that happens?